python/tk_multi_publish2/processing/plugin.py

```diff
@@ -129,6 +139,49 @@ def __init__(self, name, path, settings, logger):
 
         self._icon_pixmap = self._load_plugin_icon()
 
+    def _create_hook_instance(self, path):
+        """
+        Create the plugin's hook instance.
+
+        For publish plugins, we inject the plugin base hook's class in order to
+        provide a default implementation of the task settings UI.
+        """
+
+        # get the path to the base plugin
+        plugin_base_hook_file = os.path.join(
+            self._bundle.disk_location,
+            "hooks",
+            # don't include a .py here. it's a legacy core thing to add it
+            # automatically
+            "plugin_base"
+        )
```

Comment:
Is there a way to pass something like "{self}/hooks/plugin_base.py" to a method and get the class or module back?

Reply:
Yes! This is much cleaner now. Done!